src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -264,7 +264,7 @@
     text: |
         az acr import -n MyRegistry --source sourcerepository:sourcetag -t targetrepository:targettag \\
             -r /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/sourceResourceGroup/providers/Microsoft.ContainerRegistry/registries/sourceRegistry
-  - name: Import an image and return right after starting an import.
+  - name: Import an image without waiting for completion.
```

Comment:
@jaysterp it might be overkill, but I wonder if we should hint that this doesn't check for success.
maybe say: 
```suggestion
  - name: Import an image without waiting for successful completion.
```

Reply:
@adewaleo I agree with this suggestion. @CarolineNB